ding_place2book.install

```diff
@@ -64,6 +64,8 @@ function ding_place2book_schema() {
  * Implementation of hook_install().
  */
 function ding_place2book_install() {
+  // Set 'Maintain copy on Place2Book' as default.
```

Comment:
@cableman : Added comment. As for update hook: I don't think we can make any assumptions on the choices made (or not made) by administrators on existing installations.